README.md

```diff
@@ -5,38 +5,45 @@ English | [中文](./README-zh-CN.md)
 # OceanBase Developer Center (ODC)
 
 OceanBase Developer Center, or ODC for short, is an open-source, all-encompassing tool for collaborative database development and data management across various scenarios. By streamlining collaborative efforts, ODC addresses the challenges of risk management during database changes, efficient data management, and data security concerns.
+![image](https://github.com/oceanbase/odc/assets/15030999/cef8937e-904f-43a0-b5c6-22ebae10905a)
+
 
 ## 🤔 What ODC Offers and Why Choose ODC?
 
 ### Effortless SQL Development, Anytime, Anywhere
 
 - Utilizing modern web architecture, ODC provides unparalleled accessibility to your databases, allowing you to perform SQL development tasks on the go, right from your web browser.
 - ODC boasts a comprehensive and user-friendly SQL development environment that encompasses all the features you'd expect from desktop development tools, and even offers support for PL debugging.
+![image](https://github.com/oceanbase/odc/assets/15030999/1d9e32a6-f359-4286-b0be-82cf514c0883)
 
 
 ### Guard Every Change in Your SQL Development Journey 
 
 - Throughout the entire SQL development process, ODC is your steadfast companion, offering risk control at every turn. From visual object management to SQL queries, data editing, and import/export tasks, ODC comes equipped with built-in risk management tools.
 - ODC introduces project-based collaboration and change approval workflows, complemented by SQL validation rules, SQL window guidelines, and risk level identification.
+![image](https://github.com/oceanbase/odc/assets/15030999/6d90509b-91cb-465a-ac75-1604f96c9ffd)
```

Comment:
the backgroud color may change to white, as this is the only picture with deep color.

and, there exists one red alert, miss blank between 2 words ?
![image](https://github.com/oceanbase/odc/assets/54161870/91315a94-d92c-4830-8a61-fb0686400b12)